### PR TITLE
MVP candidate

### PR DESCRIPTION
**Pull request by itsthatriver**

### CODE REVIEW

**Comment by itsthatriver** [comment]
So this should have the basic pieces we need for MVP, please review it for correctness and style. I did note in there that this won't be the final solution for insertions to the rep table, but it should be just fine for MVP

File: functions/submit_user.js [hunk 1]
```diff
@@ -2,6 +2,41 @@
...
+
+AWS.config.update({
+  region: "us-west-2"
+});
+
+var docClient = new AWS.DynamoDB.DocumentClient();
+var userObj = {
+  TableName: "users",
+  Item: {
+    "firstName": "",
```
Comment (xavdid) [inline functions/submit_user.js]: minor, but you don't need quotes here. The standard is to do:

```javascript
{
  firstName: '',
  lastName: '',
  ...
```

File: functions/submit_user.js [hunk 2]
```diff
@@ -33,6 +80,14 @@ module.exports.handler = (event, context, callback) => {
       .filter(o => o.roles[0] === 'legislatorUpperBody')
       .map(o => o.officialIndices)[0];
 
+    representativeObj.district = data.offices[0].name.match(/[A-Z]{2}-\d\d?$/);
```
Comment (xavdid) [inline functions/submit_user.js]: I believe these all need to be `representativeObj.Item.<key>`

File: functions/submit_user.js [hunk 3]
```diff
@@ -10,12 +45,18 @@ module.exports.handler = (event, context, callback) => {
     event: JSON.stringify(event),
   });
 
+  userObj.email = event.body.email;
+  userObj.firstName = event.body.firstName || "";
+  userObj.lastName = event.body.lastName || "";
+  userObj.preference = event.body.preference || "email";
+  userObj.phoneNumber = event.body.phoneNumber || "";
```
Comment (xavdid) [inline functions/submit_user.js]: Same here, `userObj.Item.<key>`

**Review by xavdid — CHANGES_REQUESTED** [review]
Besides assigning values to the wrong key, it looks good!

If you wanted to assign things a little cleaner, you could do something like

```javascript
let user = {}

user.firstName = event.body.firstName || "";
user.lastName = event.body.lastName || "";
...

userObj.Item = user;
```

Either way though.

The other helpful change is that since it looks like we can write w/ es6 features, we should use `let` or `const` over `var`. That beings said, I believe lambda runs `v4.3.2` and there are other changes to make if we're going to use those features.

**Review by sethvoltz — APPROVED** [review]
We doin' this.

**Pull request merged**